js/src/carousel.js

```diff
@@ -197,8 +202,47 @@ class Carousel extends BaseComponent {
   }
 
   // Private
+  _removeCarouselItems() {
```

Comment:
Do you think that those 3 functions should be public ? Or maybe `checkItemsBehavior` only ?

Reply:
For sure not public ;)